lib/material_select/_mixins.scss

```diff
@@ -76,8 +76,8 @@
 ///
 /// Apply to <material-select> or <material-select-item>.
 @mixin material-select-selected-color($color) {
-  ::ng-deep .selected-accent.mixin {
-    background-color: $color;
+  ::ng-deep .selected-accent.mixin.mixin {
```

Comment:
The class selector `.mixin` is specified twice, was that intentional?

Reply:
cc @zacharydenton

Reply:
It is probably on purpose. It is a trick we use to get more specificity for mixins.